### PR TITLE
Adds a wrecked NT ship awaysite. 

### PR DESCRIPTION
**Pull request by Yonnimer**
Title^

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

**Comment by Yonnimer** [comment]
Currently broken, will fix it soonish

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

File: maps/away/generic/wrecked_nt_ship.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	id = "wrecked_nt_ship"
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
+
+/decl/submap_archetype/wrecked_nt_ship
+	map = "wrecked_nt_ship"
+	descriptor = "A wrecked ship once owned by NanoTrasen."
+
+/obj/effect/overmap/visitable/wrecked_nt_ship
+	name = "wrecked NT ship"
+	desc = "A wrecked ship once owend by NanoTrasen."
```
Comment (SleepyGemmy) [inline maps/away/generic/wrecked_nt_ship.dm:17]:
```suggestion
	desc = "A wrecked ship once owned by NanoTrasen."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
image
this tile on the shown co-ordinates is missing an area. add either one of the near areas to this tile and the unit test should pass.

**Comment by Yonnimer** [comment]
> image this tile on the shown co-ordinates is missing an area. add either one of the near areas to this tile and the unit test should pass.

Should be fixed

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**